src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -262,6 +262,7 @@ internal enum MessageID
         IDS_Missing = MessageBase + 12830,
         IDS_FeatureLambdaOptionalParameters = MessageBase + 12831,
         IDS_FeatureLambdaParamsArray = MessageBase + 12832,
+        IDS_DisableLengthBasedSwitch = MessageBase + 12833,
```

Comment:
>IDS_DisableLengthBasedSwitch [](http://example.com/codeflow?start=8&length=28)

Where is the id used?

Reply:
`WithExperimental` takes a `MessageID`. It's used in tests